src/Employer/Employer.Web/Startup.Configure.cs

```diff
@@ -1,21 +1,19 @@
 ﻿using Esfa.Recruit.Employer.Web.Configuration;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Rewrite;
 using Microsoft.Extensions.Options;
 using System.Collections.Generic;
-using System.Threading.Tasks;
 
 namespace Esfa.Recruit.Employer.Web
 {
     public partial class Startup
     {
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env,  IOptions<ExternalLinksConfiguration> externalLinks)
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, IOptions<ExternalLinksConfiguration> externalLinks)
         {
             app.UseStatusCodePagesWithReExecute("/Error/Index/{0}");

```

Comment:
I think "Index" needs to be removed here else error pages for non 500 errors won't redirect properly to custom error controller. (This issue wasn't introduced in this pull request)

Reply:
Yep, I spotted this when I was making changes. Will remove.

Reply:
I have made the amendment. 404 errors (for example) go to the correct action method on the `Error` controller now. The `ConcurrencyException` throw from the orchestrator's for editing a non-draft vacancy goes to the correct action method on the error controller as well.